README.md

```diff
@@ -40,7 +40,7 @@ projects. This tool does not only detect them, but also fixes them for you.
 
 The recommended way to install PHP CS Fixer is to use [Composer](https://getcomposer.org/download/):
 
-```console
+```sh
```

Comment:
Left is old style, right is new style:

<img width="1770" height="496" alt="Image" src="https://github.com/user-attachments/assets/2f35e316-84f3-4cfa-be20-d53038886bcf" />